CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -671,6 +652,20 @@ function Staff:afterLoad(old, new)
     return
   end
 
+  if old < 161 and new >= 161 then
```

Comment:
Never quite understood why we have `old` and `new` together. Though it seems it could prevent some issues in #1779 where debug is on (though not the intended desire either).